include/megaapi.h

```diff
@@ -6707,6 +6707,18 @@ class MegaApiLock
      */
     void lockOnce();
 
+    /**
+     * @brief Tries to lock the MegaApi if this instance does not currently have a lock on it yet.
+     *
+     * If the lock is succeeded in the expected time, the behaviour is the same as lockOnce().
+     *
+     * @param time Milliseconds to wait for locking
+     *
+     * @return if the locking succeded
+     */
+    bool try_lock_for(long long time);
```

Comment:
Can we assign a name more in line with the rest of methods in `MegaApi`?
```suggestion
    bool tryLockFor(long long time);
```